examples/ember/ember-cli-build.js

```diff
@@ -4,7 +4,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
-    // Add options here
+    'ember-welcome-page': {
```

Comment:
Out of curiosity, is this documented?

Reply:
Configuring Ember apps? Or the config options for this demo component?

Reply:
This config option.

Reply:
Not that I know of, no. The component is meant to be thrown away when you develop your app.